v0.3/handbook/libraries/stdlib.md

```diff
@@ -1400,3 +1405,55 @@ bind
     </td>
   </tr>
 </table>
+
+## System
+
+The system library provides various system-level utilities for Eve.
+
+<table class="libitem">
+  <tr>
+    <td colspan="2">
+      <b>#system/timer</b> - starts a timer at the specified resolution
+    </td>
+  </tr>
+
+  <tr>
+    <td>
+      <ul>
+        <li><strong>resultion</strong> - the frequency in milliseconds of the timer.</li>
+        <li><strong>year</strong> - (optional) - the current year</li>
+        <li><strong>month</strong> - (optional) - the current month (1 - 12)</li>
+        <li><strong>day</strong> - (optional) - the current day of the month (1 - 31)</li>
+        <li><strong>weeday</strong> - (optional) - the current day of the week (1 - 7, where 1 is Sunday)</li>
+        <li><strong>hours</strong> - (optional) - the current hour (0 - 23)</li>
+        <li><strong>minutes</strong> - (optional) - the current minute (0 - 59)</li>
+        <li><strong>seconds</strong> - (optional) - the current second (0 - 59)</li>
+        <li><strong>milliseconds</strong> - (optional) - the current millisecond (0 - 999)</li>
+        <li><strong>timestamp</strong> - (optional) - the current time represented as the number of milliseconds elapsed since 1 January 1970 00:00:00 UTC</li>
+        <li><strong>frame</strong> - (optional) - the number of frames elapsed since the start of the timer</li>
```

Comment:
We need to indicate what frame specifically means here (the number of times the timer has ticked). In fact, frame is kind of a stupid name for what it is now, I'd probably call it "tick", but I guess we shouldn't block documenting it on fixing the actual watcher.